ratatui/benches/main/text.rs

```diff
@@ -0,0 +1,49 @@
+use criterion::{BatchSize, Bencher, Criterion, criterion_group};
+use ratatui::buffer::Buffer;
+use ratatui::layout::Rect;
+use ratatui::style::Stylize;
+use ratatui::text::{Line, Text};
+use ratatui::widgets::Widget;
+
+/// Benchmark for rendering a text.
+fn text(c: &mut Criterion) {
+    let mut group = c.benchmark_group("text");
+    for (width, height) in [
+        (1, 256),   // Heavily vertically skewed
+        (256, 1),   // Heavily horizontally skewed
+        (50, 50),   // Typical rendering area
+        (100, 50),  // Vertically split screen
+        (200, 50),  // 1080p fullscreen with medium font
+        (256, 256), // Max sized area
```

Comment:
I believe we can go bigger these days as we increased .area() to u32, so this should accept u16::MAX &^ 2.